qiskit/providers/ibmq/ibmqfactory.py

```diff
@@ -73,6 +74,9 @@ def enable_account(self, token, url=QX_AUTH_URL, **kwargs):
 
         # For API 1, delegate onto the IBMQProvider.
         if not version_info['new_api']:
+            warnings.warn('IBM Experience 1 account is deprecated. '
+                          'IBMQ.update_account() for updating your stored credentials.',
```

Comment:
In this case, I think we don't want to mention `update_account()`, as `enable_account` is meant to be invoked directly (ie. without involving _stored_ credentials). Let's leave this out for the moment, and revise later.